scalardb-sample/README.md

```diff
@@ -1,162 +1,124 @@
 
-## Prerequisites
+## Overview
 
-- Java (OpenJDK 8 or higher)
-- Gradle
-- Docker, Docker Compose
-
-## Sample application
-
-### Overview
-
-This tutorial illustrates the process of creating a sample e-commerce application, where items can be ordered and paid for with a credit card using ScalarDB.
-In this tutorial, you will build the application on Cassandra.
-Although Cassandra does not provide ACID transaction capability, you can run ACID transactions on Cassandra if you interact with it through ScalarDB.
-Please note that application-specific error handling, authentication processing, and similar functions are not included in the sample application, as the focus is on demonstrating the use of ScalarDB.
-For detailed information on exception handling in ScalarDB, see [Handle SQLException](https://github.com/scalar-labs/scalardb/blob/master/docs/api-guide.md#handle-exceptions).
-
-### Schema
-
-[The schema](schema.json) is as follows:
-
-```json
-{
-  "sample.customers": {
-    "transaction": true,
-    "partition-key": [
-      "customer_id"
-    ],
-    "columns": {
-      "customer_id": "INT",
-      "name": "TEXT",
-      "credit_limit": "INT",
-      "credit_total": "INT"
-    }
-  },
-  "sample.orders": {
-    "transaction": true,
-    "partition-key": [
-      "customer_id"
-    ],
-    "clustering-key": [
-      "timestamp"
-    ],
-    "secondary-index": [
-      "order_id"
-    ],
-    "columns": {
-      "order_id": "TEXT",
-      "customer_id": "INT",
-      "timestamp": "BIGINT"
-    }
-  },
-  "sample.statements": {
-    "transaction": true,
-    "partition-key": [
-      "order_id"
-    ],
-    "clustering-key": [
-      "item_id"
-    ],
-    "columns": {
-      "order_id": "TEXT",
-      "item_id": "INT",
-      "count": "INT"
-    }
-  },
-  "sample.items": {
-    "transaction": true,
-    "partition-key": [
-      "item_id"
-    ],
-    "columns": {
-      "item_id": "INT",
-      "name": "TEXT",
-      "price": "INT"
-    }
-  }
-}
-```
+The sample e-commerce application shows how users can order and pay for items by using a line of credit.
 
-All the tables are created in the `sample` namespace.
+The database that you will be using in the sample application is Cassandra. Although Cassandra does not provide ACID transaction capabilities, you can make transactions ACID compliant by having your application connect to the database through ScalarDB.
 
-- `sample.customers`: a table that manages customers' information
-    - `credit_limit`: the maximum amount of money a lender will allow each customer to spend when using a credit card
-    - `credit_total`: the amount of money that each customer has already spent by using the credit card
-- `sample.orders`: a table that manages order information
-- `sample.statements`: a table that manages order statement information
-- `sample.items`: a table that manages information of items to be ordered
+{% capture notice--info %}
```

Comment:
Then, it might be better to add a link to the docs site in the GitHub repository.

Reply:
@choplin I agree. Since these samples apply to both Community edition and Enterprise edition, we should provide links to both in the `README.md` in the root folder. I'll figure out the best way to do that.

We should also add a link to the sidebar in this repository (under the **About** section), but let me determine the link to use first🙇🏻‍♂️